File: app/src/main/java/com/tarento/checkin/checkin/AdminCheckIn.java
package com.tarento.checkin.checkin;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class AdminCheckIn extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_admin_check_in);
    }

    public void capturePhoto(View view) {
        Intent intent = new Intent(this, CameraActivity.class);
        startActivity(intent);
        finish();
    }
}

File: app/src/main/java/com/tarento/checkin/checkin/AdminCheckIn.kt
package com.tarento.checkin.checkin

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class AdminCheckIn : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_admin_check_in)
    }

    fun capturePhoto(view: View) {
        val intent = Intent(this, CameraActivity::class.java)
        startActivity(intent)
        finish()
    }
}
